packages/image_picker/image_picker_platform_interface/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ repository: https://github.com/flutter/plugins/tree/master/packages/image_picker
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.4.0
+version: 2.3.0
```

Comment:
You can't revert the version, because 2.3.0 and 2.4.0 are already published. This should be 2.4.1.

Reply:
Done.